client/db/types.go

```diff
@@ -257,6 +257,21 @@ type OrderMetaData struct {
 	// RedemptionFeesPaid is the sum of the actual fees paid for all
 	// redemptions.
 	RedemptionFeesPaid uint64
+
+	// NOTE: We probably should be storing any variable information of the
+	// dex.Asset (the server's asset config at time of order). We store the max
+	// fee rates for swap and redeem, and the asset versions, but the required
+	// swap confirmations could change too:
+	//
+	// FromSwapConf uint64
+	// ToSwapConf   uint64
+	//
+	// Without EpochIdx stored explicitly, we currently infer based on the epoch
+	// duration in the *current* market config (actually variable) and
+	// the order's ServerTime stamp. It might be wise to record it too:
+	//
+	// EpochIdx     uint64
```

Comment:
Before bringing out of draft, I'm going to go forward with storing these data in the client DB, except I'm going to store epoch **duration** instead of index.